src/components/Form.js

```diff
@@ -1,15 +1,33 @@
-import React from 'react';
+import { useSelector } from 'react-redux';
+import { React, useState } from 'react';
 import './Form.css';
+import store from '../redux/configureStore';
+import { addBK } from '../redux/books/books';
 
-const Form = () => (
+const Form = () => {
+  const dataBook = useSelector((state) => state.books);
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const titleInput = (e) => { setTitle(e.target.value); };
+  const authorInput = (e) => { setAuthor(e.target.value); };
 
-        <form className="form">
-         <h1 className="addNbook">ADD NEW BOOK</h1>
-    <input type='text' className="inputBook" placeholder="Title"></input>
-    <input type='text' className="inputAuthor" placeholder="Author"></input>
+  return (
+      <>
+      <form className="form">
 
-  <button className="addBook">ADD BOOK</button>
+    <input type='text' onInput={ titleInput } className="inputBook" placeholder="Title" value={title}></input>
+    <input type='text' onInput={ authorInput } className="inputAuthor" placeholder="Author" value={author}></input>
+
+  <button className="addBook" onClick={(e) => {
+    e.preventDefault();
+    if (title !== '' && author !== '') {
+      store.dispatch(addBK({ id: dataBook.length + 1, title, author }));
+      setAuthor('');
+      setTitle('');
```

Comment:
- [ ] @J2ZROMERO great work on your component, it would be better if you created a method to pass the functionality of adding a new book and then call the method on the onClick method instead of writing the logic here, it would make your code cleaner.
`hint` ``just like you did in the book component``